packages/fxa-content-server/app/scripts/views/mixins/third-party-auth-mixin.js

```diff
@@ -70,6 +71,8 @@ export default {
 
     this.logFlowEvent('google.oauth-start');
 
+    GleanMetrics.emailFirst.googleOauthSubmit();
```

Comment:
We should be able to do a check on the view name  `this.viewName === 'email-first'`, to emit the correct version.